src/test/java/org/folio/rest/impl/SynchronizationAPITests.java

```diff
@@ -160,6 +170,41 @@ public void declaredLostEventShouldBeCreatedAfterSynchronization() {
     checkSyncJobUpdatedByLoanEvent(syncJobId);
   }
 
+  @Test
+  public void agedToLostEventShouldBeDeletedBeforeSynchronizationJobByUser() {
+    sendAgedToLostEvent(createItemAgedToLostEvent(USER_ID), SC_NO_CONTENT);
+
+    stubLoans(now().plusHours(1).toDate(), true, "Checked out");
```

Comment:
Why do we need stub loans at all here, does this loan affect the result? Maybe we can use stubLoansWithEmptyResponse()?

Reply:
because I use checkSyncJobUpdatedByLoanEvent so I added loan stubs,
```
protected void checkSyncJobUpdatedByLoanEvent(String syncJobId) {
    Awaitility.await()
      .atMost(30, SECONDS)
      .until(() -> waitFor(synchronizationJobRepository.get(syncJobId))
        .orElse(null), is(synchronizationJobMatcher(JOB_STATUS_DONE, 0, 0, 1, 0)));
  }
```
and I just want to use existing check sync job function, then now I added my own it just checks for the job with empty values.